gradle/libs.versions.toml

```diff
@@ -0,0 +1,9 @@
+[plugins]
+detekt = "io.gitlab.arturbosch.detekt:1.19.0"
+kotlin = "org.jetbrains.kotlin.kapt:1.6.10"
+ktlint = "org.jlleitschuh.gradle.ktlint:10.2.1"
+pluginPublish = "com.gradle.plugin-publish:0.14.0"
+versionCheck = "com.github.ben-manes.versions:0.42.0"
```

Comment:
Can you export versions?

Reply:
7c324a4c48bc05458084f68a679df6f5ff4cf1c4